app/assets/stylesheets/ama_layout/layout_components/button-grouping.scss

```diff
@@ -22,6 +22,13 @@
       min-height: 170px;
       position: relative;
       width: 100%;
+
+      &--green{
+        @extend .button;
+        background-color: $brand-green;
+        width: 100%;
+        padding: 45px 0 45px 0;
```

Comment:
does this padding need to be more than it is everywhere else? or can we just use padding: $base-padding instead?